data/autoyast_hanaperf/agama-config.jsonnet

```diff
@@ -63,24 +63,13 @@
         chroot: true,
         body: |||
           #!/usr/bin/env bash
-          HOST_INFO_DB=(
-          "hana01","2c:ea:7f:ea:b0:24"
-          "hana02","2c:ea:7f:ea:ad:0c"
-          "hana03","2c:ea:7f:ea:bd:7c"
-          "hana04","5c:6f:69:14:14:12"
-          "hana05","5c:6f:69:13:f3:84"
-          )
-          for HOST_INFO in ${HOST_INFO_DB[@]}; do
-                IFS=',' read -r HOST_SHORT_NAME HOST_MAC <<< "${HOST_INFO}"
-                if ip link show | grep -qi "${HOST_MAC}";then
-                    break
-                fi
-          done
           echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf
+          # Workaround for sshd service deactivated bsc#1238590
+          systemctl enable sshd
           # Workaround for NetworkManager to make sure the expected NIC up only
           rm -f /etc/NetworkManager/system-connections/default_connection.nmconnection
           echo -e "[main]\nno-auto-default=type:ethernet" > /etc/NetworkManager/conf.d/disable_auto.conf
-          echo -e "[connection]\nid=nic0\nuuid=$(uuidgen)\ntype=ethernet\n[ethernet]\nmac-address=${HOST_MAC}\n[ipv4]\nmethod=auto\n" > /etc/NetworkManager/system-connections/nic0.nmconnection
+          echo -e "[connection]\nid=nic0\nuuid=$(uuidgen)\ntype=ethernet\n[ethernet]\nmac-address={{SUT_NETDEVICE}}\n[ipv4]\nmethod=auto\n" > /etc/NetworkManager/system-connections/nic0.nmconnection
```

Comment:
AFAIR, `SUT_NETDEVICE` is mostly used in os-autoinst-distri-opensuse set to the name of the network interface used in the SUT (`eth0` for example). That sounds different than a mac address. See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/boot/boot_from_pxe.pm#L26 for example. For the mac address, the setting most commonly used is `NICMAC`, it's even used in some of the backends such as qemu: https://github.com/os-autoinst/os-autoinst/blob/master/backend/qemu.pm#L773.

I propose to change it if possible.

Reply:
Thank you, we usually overwrite the default value (eth0) of `SUT_NETDEVICE` by mac address for bare-metal installation, especially there are many NIC ports in bare-metal, their order is not consistent, eth0 usually points to different mac address, that causes confision during installation, for example: different hostname.  Using SUT_NETDEVICE is same to inherit existed parameter. 
May I create a new variable to store that? Thanks!

Reply:
No. I think it's OK.